test/videojs.wavesurfer.spec.js

```diff
@@ -275,78 +296,64 @@ describe('Wavesurfer', () => {
             });
         });
     });
+    */
 
     /** @test {Wavesurfer#load} */
     it('loads peaks from array', (done) => {
 
-        player.one(Event.WAVE_READY, () => {
-            player.wavesurfer().surfer.once(Event.REDRAW, done);
-            // load with peaks data
-            player.wavesurfer().load(
-                TestHelpers.EXAMPLE_AUDIO_FILE,
-                [
-                    -0.007874015748031496, 0.0, -0.007874015748031496,
-                    0.0, -0.007874015748031496, 0.0, -0.007874015748031496,
-                    0.0, -0.007874015748031496, 0.0, -0.007874015748031496,
-                    0.0, -0.007874015748031496, 0.0, -0.007874015748031496,
-                    0.0, -0.007874015748031496, 0.0, -0.007874015748031496
-                ]
-            );
+        player.one(Event.WAVE_READY, done);
+
+        player.src({
+            src: TestHelpers.EXAMPLE_AUDIO_FILE,
+            type: TestHelpers.EXAMPLE_AUDIO_TYPE,
+            peaks: [
+                -0.007874015748031496, 0.0, -0.007874015748031496,
+                0.0, -0.007874015748031496, 0.0, -0.007874015748031496,
+                0.0, -0.007874015748031496, 0.0, -0.007874015748031496,
+                0.0, -0.007874015748031496, 0.0, -0.007874015748031496,
+                0.0, -0.007874015748031496, 0.0, -0.007874015748031496
+            ]
         });
     });
 
     /** @test {Wavesurfer#load} */
-    it('loads peaks from file', (done) => {
+    it('loads peaks from JSON file', (done) => {
 
-        player.one(Event.WAVE_READY, () => {
-            player.wavesurfer().surfer.once(Event.REDRAW, done);
-            // load from peaks file
-            player.wavesurfer().load(
-                TestHelpers.EXAMPLE_AUDIO_FILE,
-                TestHelpers.EXAMPLE_AUDIO_PEAKS_FILE
-            );
-        });
+        player.one(Event.WAVE_READY, done);
+
+        player.src({
+            src: TestHelpers.EXAMPLE_AUDIO_FILE,
+            type: TestHelpers.EXAMPLE_AUDIO_TYPE,
+            peaks: TestHelpers.EXAMPLE_AUDIO_PEAKS_FILE
+        })
```

Comment:
I was trying to build this locally, but it looks like this is missing a semicolon, so the linter fails. Can this be fixed (no rush)? 🙂

Reply:
you can always make a pull request to fix it ;)

I fixed it for you.